### ISSUE TITLE
Tag unsupported error on oneof usage 

### ISSUE THREAD

**Issue by jubyvictor** [bxcodec/faker#108]
**Issue:**
Unable to use the oneof tag

**Steps to reproduce.** 

1) Create a struct that looks like this.
```

import(
  "fmt"
)
type Engagement struct {
	IsEntity   bool   `default:"True"`
	EventType  string `faker:"oneof: a, b"`
}
```

Try printing it
```

func GenerateEngmt() {
	data := Engagement{}
	err := faker.FakeData(&data)
	if err != nil {
		fmt.Printf("ERR %v\n", err)
	}
	fmt.Printf("%v", data)
}
```

**Output**

`ERR Tag unsupported: oneof: a, b`

**Go env**
```
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/wolf/Library/Caches/go-build"
GOENV="/Users/wolf/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/wolf/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/Users/wolf/Tools/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/Users/wolf/Tools/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/wolf/Work/utils/load-ebus-go/ebuspub/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/cq/35z51m017zs1zv8n_jcnc5x9kwcqwm/T/go-build970742704=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

**Comment by bxcodec** [comment]
Hi @jubyvictor 

May I know your faker version? 
oneOf is not released yet, to enable it you can use the master version. 

```
go get github.com/bxcodec/faker/v3@master
```

**Comment by jubyvictor** [comment]
Ah cool, that explains. Apologies should have mentioned this. Running 3.4.0

**Comment by filipegorges** [comment]
Hi @bxcodec, might be worth mentioning this on the example as well (https://github.com/bxcodec/faker/blob/master/example_with_tags_test.go)

**Comment by bxcodec** [comment]
This issue should be fixed already in the new version, v3.5.0 https://github.com/bxcodec/faker/releases/tag/v3.5.0
